docs/products/access/api-reference/config-api/webhooks-configuration-api.md

```diff
@@ -250,10 +248,78 @@ In the event of an error, one of the [generic error codes](#error-codes) will be
+| 400         | invalid_request | One or more parameters is missing or incorrect. The details contain the missing or incorrect parameters. |
+| 401         | unauthorized    | Provide valid credentials to get access to the API.                                                      |
+| 403         | forbidden       | Operation is not allowed for the current user.                                                           |
+| 404         | not_found       | Web Hook configuration cannot be found for this id                                                       |
+| 409         | conflict        | The id already exists for a different Web Hook configuration                                             |
+
+# Authentication Methods
+
+## JWT
+
+JSON Web Tokens (JWT) is the preferred method for authentication.
+JWT is a reliable, secure method of representing claims to be transferred between two parties.
+
+Example JWT that Access generates for use in WebHooks:
+
+```json
+{
+  "aud": "http://wiremock:8080",
+  "sub": "onewelcomeAccessWebHookClient",
+  "scope": [
+    "onewelcome_webhooks",
+    "onewelcome_webhook_test"
+  ],
+  "iss": "https://playground.test.onewelcome.io/oauth",
+  "exp": 1705585175,
+  "jti": "AT.4027b2b3-b3f8-4ff9-bef7-c7a0f01d6bec",
+  "cid": "onewelcomeAccessWebHookClient"
+}
+```
+- `aud` (Audience): Specifies the recipients that the JWT is intended for. It will contain `base_uri` from WebHook configuration
+- `sub` (Subject): Identifies the principal that is the subject of the JWT. In this case, it represents the identity of the webhook client
+  named `onewelcomeAccessWebHookClient`.
+- `scope`: Defines the permission scope granted to the WebHook client. It will contain two scopes a fixed one `onewelcome_webhooks`
+  and one based on the name of the WebHook - `onewelcome_webhook_<name>` where `<name>` will be replaced with lower cased WebHook name
+  where space characters are replaced with underscore sign. For example, WebHook named `My WebHook` will contain
+  scope `onwewelcome_webhook_my_webhook`.
+- `iss` (Issuer): Identifies the principal that issued the JWT. It will contain URL of Access service which calls the WebHook.
+- `exp` (Expiration Time): Specifies the expiration time after which the JWT must not be accepted for processing. It is a numeric value
+  representing the number of seconds from 1970-01-01T00:00:00Z UTC until the specified UTC date/time.
+- `jti` (JWT ID): Provides a unique identifier for the JWT, which can be used to prevent the JWT from being replayed. It is a case-sensitive
+  string.
+- `cid` (Client ID): Identifies the client that requested the JWT. It will contain a fixed value `onewelcomeAccessWebHookClient`.
+
+Token will be signed by Access Service and can be verified by using public keys available
+at: `https://<access_service_hostname>/oauth/v1/keys`
```

Comment:
instead of using `<access_service_hostname>` we can maybe use the `iss` claim to explain how the URL is build up.